<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerControls</class>
 <widget class="QWidget" name="PlayerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="player_controls_layout">
     <item>
      <widget class="QPushButton" name="player_controls_previous">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/previous.png</normaloff>:/controls/controls/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="player_controls_play">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/play.png</normaloff>:/controls/controls/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="player_controls_stop">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/stop.png</normaloff>:/controls/controls/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="player_controls_next">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/next.png</normaloff>:/controls/controls/next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="player_controls_speed">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/random.png</normaloff>:/controls/controls/random.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="player_controls_play_mode">
       <property name="minimumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>33</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/controls/controls/loop.png</normaloff>:/controls/controls/loop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
